src/coreclr/jit/lclvars.cpp

```diff
@@ -1678,6 +1671,10 @@ void Compiler::lvaClassifyParameterABI(Classifier& classifier)
 
         lvaParameterPassingInfo[i] = classifier.Classify(this, dsc->TypeGet(), structLayout, wellKnownArg);
 
+#if FEATURE_IMPLICIT_BYREFS
+        dsc->lvIsImplicitByRef = lvaParameterPassingInfo[i].IsPassedByReference();
+#endif // FEATURE_IMPLICIT_BYREFS
```

Comment:
I have found in the past `Marking V%02i as a byref parameter` from below to be a useful crumb to have a quick understanding of which parameters are by-ref. Preserve it by moving here?

Reply:
It currently gets printed by `ABIPassingInformation::Dump`, so at the top of the JIT dump you will see:

```csharp
static void Foo(long a, object b, Guid c, List<string>.Enumerator d)
{
}
```

```scala
Parameter V00 ABI info: [00..08) reg x0
Parameter V01 ABI info: [00..08) reg x1
Parameter V02 ABI info: 2 segments
  [0] [00..08) reg x2
  [1] [08..16) reg x3
Parameter V03 ABI info: [00..08) reg x4 (implicit by-ref)
```

Let me know what you think.

Reply:
LGTM.